client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -90,176 +90,189 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
-                        />
-                    }
-                />
-                <Autocomplete
-                    options={streetsInCity}
-                    getOptionLabel={(option) => option.displayName}
-                    inputValue={isolationStreetName}
-                    onChange={(event, selectedStreet) => updateIsolationAddress(ClinicalDetailsFields.ISOLATION_STREET, selectedStreet === null ? '' : selectedStreet.id)}
-                    onInputChange={(event, selectedStreetName) => {
-                        setIsolationStreetName(selectedStreetName);
-                        if (selectedStreetName === '') {
-                            updateIsolationAddress(ClinicalDetailsFields.ISOLATION_STREET, '');
-                        }
-                    }}
-                    renderInput={(params) =>
-                        <TextField
-                            test-id='currentQuarantineStreet'
-                            {...params}
-                            placeholder='רחוב'
-                            className={classes.cityStreetTextField}
-                        />
-                    }
-                />
-                <AlphanumericTextField
-                    test-id='currentQuarantineHomeNumber'
-                    name={ClinicalDetailsFields.ISOLATION_HOUSE_NUMBER}
-                    value={context.clinicalDetailsData.isolationAddress.houseNum}
-                    onChange={(newValue: string) => (
-                        updateIsolationAddress(ClinicalDetailsFields.ISOLATION_HOUSE_NUMBER, newValue)
-                    )}
-                    setError={setError}
-                    clearErrors={clearErrors}
-                    errors={errors}
-                    placeholder='מספר הבית'
-                    className={classes.floorHouseNumTextField}
-                />
-                <AlphanumericTextField
-                    test-id='currentQuarantineFloor'
-                    name={ClinicalDetailsFields.ISOLATION_FLOOR}
-                    value={context.clinicalDetailsData.isolationAddress.floor}
-                    onChange={(newValue: string) => (
-                        updateIsolationAddress(ClinicalDetailsFields.ISOLATION_FLOOR, newValue)
-                    )}    
-                    setError={setError}
-                    clearErrors={clearErrors}
-                    errors={errors}
-                    placeholder='קומה'
-                    className={classes.floorHouseNumTextField}
                     />
-                <Grid item xs={12}>
                 </Grid>
-                <Grid item xs={2}>
+            </Collapse>
+            <Grid spacing={3} container className={classes.containerGrid} justify='flex-start' alignItems='center'>
+                <Grid item xs={2} className={classes.fieldLabel}>
                     <Typography>
                         <b>
-                            האם בעייתי לקיים בידוד:
+                            כתובת לאשפוז ביתי:
```

Comment:
![image](https://user-images.githubusercontent.com/42100027/94040950-42914e80-fdd2-11ea-8e70-853554b5d1f3.png)
addres fields overlap in ckinical details tab